features.js

```diff
@@ -1,19 +1,17 @@
 (async () => {
   'use strict';
 
-  function partitionArray(arr, condition) {
-    const matched = [];
-    const unmatched = [];
-
-    for (const item of arr) {
-      if (condition(item)) {
-        matched.push(item);
-      } else {
-        unmatched.push(item);
+  function partitionByPhase(features) {
+    const map = [];
+
+    for (const feature of features) {
+      if (!map[feature.phase]) {
+        map[feature.phase] = [];
       }
+      map[feature.phase].push(feature);
     }
 
-    return { matched, unmatched };
+    return map;
   }
```

Comment:
The JS changes I made have not been tested btw, not sure how well the CI works here.

Reply:
I built and verified it locally, so this is no longer a concern.